docker/test/stateless_unbundled/Dockerfile

```diff
@@ -1,12 +1,56 @@
 # docker build -t yandex/clickhouse-stateless-unbundled-test .
-FROM yandex/clickhouse-test-base
+FROM ubuntu:20.04
 
 ARG odbc_driver_url="https://github.com/ClickHouse/clickhouse-odbc/releases/download/v1.1.4.20200302/clickhouse-odbc-1.1.4-Linux.tar.gz"
 
-RUN apt-get --allow-unauthenticated update -y \
-    && env DEBIAN_FRONTEND=noninteractive \
-        apt-get --allow-unauthenticated install --yes --no-install-recommends \
-            alien \
+ENV DEBIAN_FRONTEND=noninteractive LLVM_VERSION=10
+
+RUN apt-get update \
+    && apt-get install ca-certificates lsb-release wget gnupg apt-transport-https \
+        --yes --no-install-recommends --verbose-versions \
+    && export LLVM_PUBKEY_HASH="bda960a8da687a275a2078d43c111d66b1c6a893a3275271beedf266c1ff4a0cdecb429c7a5cccf9f486ea7aa43fd27f" \
+    && wget -O /tmp/llvm-snapshot.gpg.key https://apt.llvm.org/llvm-snapshot.gpg.key \
+    && echo "${LLVM_PUBKEY_HASH} /tmp/llvm-snapshot.gpg.key" | sha384sum -c \
+    && apt-key add /tmp/llvm-snapshot.gpg.key \
+    && export CODENAME="$(lsb_release --codename --short | tr 'A-Z' 'a-z')" \
+    && echo "deb [trusted=yes] http://apt.llvm.org/${CODENAME}/ llvm-toolchain-${CODENAME}-${LLVM_VERSION} main" >> \
+        /etc/apt/sources.list
+
+# initial packages
+RUN apt-get update \
+    && apt-get install \
+        bash \
+        fakeroot \
+        ccache \
+        curl \
+        software-properties-common \
+        --yes --no-install-recommends
+
+# Special dpkg-deb (https://github.com/ClickHouse-Extras/dpkg) version which is able
+# to compress files using pigz (https://zlib.net/pigz/) instead of gzip.
+# Significantly increase deb packaging speed and compatible with old systems
+RUN curl -O https://clickhouse-builds.s3.yandex.net/utils/1/dpkg-deb \
+    && chmod +x dpkg-deb \
+    && cp dpkg-deb /usr/bin
+
+
+RUN apt-get update \
+    && apt-get install \
+        clang-${LLVM_VERSION} \
+        debhelper \
+        devscripts \
+        gdb  \
+        git \
+        gperf \
+        lcov \
+        llvm-${LLVM_VERSION} \
+        moreutils \
+        perl \
+        perl \
+        pigz \
+        pkg-config \
+        tzdata \
+                    alien \
```

Comment:
Alien does not make sense for unbundled build, because we don't ship any packages except deb for this "exotic" build variant.